js/ui/html_editor/utils/image_uploader_helper.js

```diff
@@ -63,6 +69,22 @@ export class ImageUploader {
             });
     }
 
+    getCurrentTab() {
+        return this.tabs[this.tabPanelIndex];
+    }
+
+    updateAddButtonState() {
+        const isDisabled = this.getCurrentTab().isDisableButton();
+
+        this.setAddButtonDisabled(isDisabled);
+    }
```

Comment:
Minor comment.

Maybe, should we use the arrow function here to forget about the bind method?

```suggestion
    updateAddButtonState = () => {
        const isDisabled = this.getCurrentTab().isDisableButton();
        this.setAddButtonDisabled(isDisabled);
    };
```

If we do it, we can remove the bind from the [109 line](https://github.com/DevExpress/DevExtreme/pull/23313/files#diff-335430ff6be5ce2d4092dc60ac3fdea39a9520d6d0009d650ccd48c826217f45R109)

```ts
    createUrlTab(formData) {
        return new UrlTab(this.module, {
            config: this.config,
            formData,
            isUpdating: this.isUpdating
        }, this.updateAddButtonState); // <-- here
```